model_utils/fields.py

```diff
@@ -5,7 +5,7 @@
 from django.db import models
 from django.conf import settings
 from django.core.exceptions import ValidationError
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```

Comment:
this will not needed as support for python 2 wil be dropped

Reply:
I see, though I wasn't trying to drop python 2 support at the same time, just trying to solve the Django 3-alpha stuff.

I noticed you closed the PR. Is there something you would like me to do differently?

Reply:
@auvipy ok, is there a different PR that's handling dropping Python2 or adding Django3 that I can help with?